Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -797,19 +921,34 @@ TextureCacheBase::TCacheEntry* TextureCacheBase::Load(const u32 stage)
   else if (!hires_tex)
   {
     size_t decoded_texture_size = expandedWidth * sizeof(u32) * expandedHeight;
-    CheckTempSize(decoded_texture_size);
+
+    //allocate memory for all levels at once
```

Comment:
Spaces after `//` indicators.

Reply:
Do you have a version of the linter that runs offline? I don't wanna keep wasting your time with this stuff :P

Reply:
Yeah, we use clang-format. Our config file for it can be found [here](https://github.com/dolphin-emu/dolphin/blob/master/Source/.clang-format)

Reply:
Tools/lint.sh

Reply:
That's great, thanks! I hope that will be the end of the style errors :P 

PS: maybe it could help to put "use this tool" on the Pull Request template, I had no idea!